docker/deployed/testnet/api/config.json

```diff
@@ -46,7 +46,8 @@
         "Enabled": true,
         "MetricsHubURL": "https://metricshub-testnet-mrgr43cf5q-uw.a.run.app",
         "MetricsHubApiKey": "${METRICS_HUB_API_KEY}",
-        "PublishingInterval": "10s"
+        "PublishingInterval": "10s",
+        "ChainStackCollectFrequency": "15m"
```

Comment:
The state-hash "frequency" is hand-wavy and not strictly constant for many reasons:
- The frequency is defined by "# blocks' and is a per-chain configuration, not global.
- The "30 mins" is an indirect result of the values we baked today, but I'm not sure how that will play out in the future.
- More fundamentally, remember that calculating the state hash of every X blocks (where X is today calculated to be ~30min with some estimations), is a hand-wavy frequency (not fixed) since also depends when a transaction happens in that chain. e.g: if a chain doesn't have a transaction in 80mins, then no state-hash will be calculated since that is triggered "on the next txn that happens after block %X=0".

That was my thought process. The last point might be the most annoying one, since we wouldn't have any metric with a fixed frequency to also infer uptime.

I think it would be totally fine to merge both, but we should be aware of this "irregular heartbeat".